docs/commands/bootstrap.mdx

```diff
@@ -7,48 +7,43 @@ description: Learn more about the `bootstrap` command in Melos.
 
 <Info>Supports all [Melos filtering](/filters) flags.</Info>
 
-This command initializes the workspace, links local packages together and
-installs remaining package dependencies.
+This command initializes the workspace and installs remaining package
+dependencies.
 
 ```bash
 melos bootstrap
 # or
 melos bs
 ```
 
-Bootstrapping has two primary functions:
+Bootstrapping has three primary functions:
 
 1. Installing all package dependencies (internally using `pub get`).
-2. Locally linking any packages together via path dependency overrides _without
-   having to edit your pubspec.yaml_.
+2. Syncing shared dependencies between packages.
+3. Running any bootstrap lifecycle scripts.
 
 ## Why is bootstrapping required?
 
-In normal projects, packages can be linked by providing a `path` within the
-`pubspec.yaml`. This works for small projects however presents a problem at
-scale. Packages cannot be published with a locally defined path, meaning once
-you're ready to publish your packages you'll need to manually update all the
-packages `pubspec.yaml` files with the versions. If your packages are also
-tightly coupled (dependencies of each other), you'll also have to manually check
-which versions should be updated. Even with a few of packages this can become a
-long and error-prone task.
-
-Melos solves this problem by overriding local files which the Dart analyzer uses
-to read packages from. If a local package exists (defined in the `melos.yaml`
-file) and a different local package has it listed as a dependency, it will be
-linked regardless of whether a version has been specified.
+After the [Pub Workspaces feature](https://dart.dev/tools/pub/workspaces) was
+introduced in Dart 3.6.0, it is no longer strictly necessary to run `melos
+bootstrap`, since all the packages are already linked together. However, there
+are still some benefits to running `melos bootstrap`, because you need to run
+`pub get` in each package to initialize the workspace, and `melos bootstrap`
+will do that for you.
```

Comment:
> because you need to run `pub get` in each package to initialize the workspace

@spydon I've just been running `dart pub get` in any folder (in a non-Melos workspace) and it initializes the whole workspace. That makes sense to me because the packages have a shared resolution.

Reply:
Might not a matter of interpretation but this paragraphs reads to me as: 

> Once you start using Pub Workspaces, you don't _need_ to run `melos bootstrap`, but it's still convenient since it will run `dart pub get` in each sub-package for you

(It does say "each package in the _workspace" too!)

Reply:
Missed you second comment!
Yeah, I guess that we should clarify that on the intro page of the docs, is it something you'd like to PR? :)

Reply:
Ah, I don't actually know the details of melos -- that's why I came here, was hoping to find such a list!